joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/ClassTests.scala

```diff
@@ -363,9 +363,9 @@ class ClassTests extends RubyCode2CpgFixture {
         |""".stripMargin)
 
     "generate a type decl with the associated members" in {
-      inside(cpg.typeDecl.nameExact("<anon-class-0>").l) {
+      inside(cpg.typeDecl.nameExact("Test0.rb:<main>.<anon-class-0>").l) {
```

Comment:
In other tests you use `$Main` instead of `<main>` please do so here as well as in the rest of this file.